packages/react-shortcuts/src/Shortcut/hooks.ts

```diff
@@ -11,23 +11,22 @@ export interface Options {
   held?: HeldKey;
   node?: HTMLElement | null;
   ignoreInput?: boolean;
+  ignoreTags?: HTMLElement['tagName'][];
```

Comment:
For context, this is needed to support undo/redo using keyboard shortcuts on Web's `Editor` component which uses a `textarea` under the hood.

Reply:
That's the idea. We are providing our editor's `textarea` as `node` and making sure `TEXTAREA` is removed from the list of ignored tags so we can hit this code path in `ShortcutManage.tsx`:

```

        if (node) {
          const onFocusedNode = document.activeElement === node;
          return partiallyMatching && onFocusedNode;
        }
```